moped-database/views/project_list_view.sql

```diff
@@ -1,6 +1,4 @@
--- latest version 1697768359465_dedupe_plv_tks
-DROP VIEW project_list_view;
```

Comment:
Should this stay? I dropped [some comments](https://github.com/cityofaustin/atd-moped/pull/1191#discussion_r1381881855) in John's PR for some context. I reviewed that one first.

Reply:
Its not necessary for this view to get updated, and if I try dropping the view first, I get errors and we would need to drop the other views and recreate them too